sys/linux/sys.txt

```diff
@@ -78,7 +78,7 @@ type signalnoptr intptr[0:65]
 
 syz_execute_func(text ptr[in, text[target]])
 
-openat$sysfs(fd const[AT_FDCWD], dir ptr[in, glob["/sys/**/*"]], flags flags[open_flags], mode flags[open_mode]) fd
+openat$sysfs(fd const[AT_FDCWD], dir ptr[in, glob["/sys/**/*:-/sys/power/state"]], flags flags[open_flags], mode flags[open_mode]) fd
```

Comment:
Please add a comment with a link to the bug as to why we remove /sys/power/state.